modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -549,21 +549,34 @@
 
      /**
      * Since server startup, whenever this interval elapses, the expired messages will be cleared from the store.
-      * This messageExpiration configuration was disabled in the broker.xml configuration file. This is for future use.
+      * specified in seconds.
      */
     PERFORMANCE_TUNING_MESSAGE_EXPIRATION_CHECK_INTERVAL
-            ("performanceTuning/messageExpiration/checkInterval", "10000", Integer.class),
+            ("performanceTuning/messageExpiration/checkInterval", "900", Integer.class),
+
```

Comment:
this is in seconds (specified in comments) . Default time would be 15 minutes. We need to decide on a suitable value